### ISSUE TITLE
Pipeline events tab redesign

### ISSUE THREAD

**Issue by tkyi** [screwdriver-cd/screwdriver#1886]
**What happened**: It's often hard to follow the progression of a commit through the pipeline, especially if the pipeline broke at some point and changes were made or jobs were rerun. Currently, each time any job with any sha is triggered manually, by a commit, or by an upstream build, a new event appears on the right side Events tab.

Multiple entries with sha #6e0dd13 (current):
![Screen Shot 2019-12-18 at 3 15 05 PM](https://user-images.githubusercontent.com/3230529/71131367-59eb3d80-21a9-11ea-8eab-1f68798a20aa.png)

**What you expected to happen**: It would be nice to have something similar to an aggregated view. Intuitively, users expect a restarted event at a particular sha to be a restart of the same event, not a brand new event. Users might expect to see the right side bar of events to have a single entry per sha with the latest build statuses per job displayed.

Single entry for each sha with aggregated view of latest builds (expected):
![Screen Shot 2019-12-18 at 3 17 20 PM](https://user-images.githubusercontent.com/3230529/71131662-2c52c400-21aa-11ea-92c7-20ee105c90c4.png)

**How to reproduce it**: Run a pipeline. Run it again manually by selecting the event and using the dropdown start option a few more times. You can see multiple entries on the right side tab with the same sha under Events.

**Comment by jithine** [comment]
This might be addressed with https://github.com/screwdriver-cd/screwdriver/issues/1820
